src/memjs/memjs.ts

```diff
@@ -626,21 +615,21 @@ class Client<Value, Extras> {
     if (callback === undefined && options !== "function") {
       if (!options) options = {};
       return promisify((callback) => {
-        this.add(
-          key,
-          value,
-          options as { expires?: number },
-          function (err, success) {
-            callback(err, success);
-          }
-        );
+        this.add(key, value, options as { expires?: number }, function (
+          err,
+          success
+        ) {
+          callback(err, success);
+        });
       });
     }
     const logger = this.options.logger;
 
     // TODO: support flags, support version (CAS)
     this.incrSeq();
-    const expiration = makeExpiration((options || {}).expires || this.options.expires);
+    const expiration = makeExpiration(
+      (options || {}).expires || this.options.expires
```

Comment:
this is on modern TS so we can do options?.expires now, forgot I think when I wrote this

Reply:
TODO. I want to get to this in the next PR where I remove callbacks!